addons/im_livechat/static/src/legacy/models/public_livechat.js

```diff
@@ -180,9 +169,11 @@ const PublicLivechat = Class.extend(Mixins.EventDispatcherMixin, {
      */
     getTypingMembersToText() {
         const typingPartnerIDs = this._typingPartnerIDs;
-        const typingMembers = _.filter(this._members, function (member) {
-            return _.contains(typingPartnerIDs, member.id);
-        });
+        const typingMembers = (
+            this.messaging.publicLivechatGlobal.publicLivechat.operator && this._typingPartnerIDs.includes(this.messaging.publicLivechatGlobal.publicLivechat.operator.id)
+            ? [this.messaging.publicLivechatGlobal.publicLivechat.operator]
+            : []
+        );
         const sortedTypingMembers = _.sortBy(typingMembers, function (member) {
             return _.indexOf(typingPartnerIDs, member.id);
         });
```

Comment:
Looking at this piece of code, it seems like there will always be only one typing partner, yet the code has been designed to handle several, I feel like there is something wrong with your changes.

Reply:
It was designed for many members, but actually `this._members` only had the operator in it.
I removed `this._members`, so here I adapted the code. Since members had only operator, that's why I changed the code with just operator in mind.
The resulting behaviour of code should be unchanged by this change